maas/cinder_service_check.py

```diff
@@ -56,6 +56,11 @@ def check(auth_ref, args):
     if len(services) == 0:
```

Comment:
I think the patch needs to go before this line, as services can be filtered down to 0 in the patch.